test/Conversion/LinalgToXsmm/linalg-to-brgemm.mlir

```diff
@@ -393,15 +401,17 @@ func.func @vnni_brgemm_require_transpose_on_C(%arg0: memref<16x32x32xbf16>, %arg
 
 // -----
 
-#map = affine_map<(d0, d1, d2, d3, d4) -> (d0, d2, d4)>
-#map1 = affine_map<(d0, d1, d2, d3, d4) -> (d0, d4 floordiv 5, d3, d1)>
+#map = affine_map<(d0, d1, d2, d3, d4) -> (d0, d2, d4, d1)>
+#map1 = affine_map<(d0, d1, d2, d3, d4) -> (d0, d4, d3, d1)>
 #map2 = affine_map<(d0, d1, d2, d3, d4) -> (d3, d2)>
```

Comment:
TODO: fix `map2` to not have transpose, change example VNNI factor to 5

Reply:
Fixed this and the equivalent test case in `linalg-to-gemm.mlir`.

After changes, this example would trigger an assertion in pass matcher. Thus, tweaked the pass logic to fail gracefully and simply leave operation unchanged.